argo/kfp-compiler/kfp_compiler/compiler.py

```diff
@@ -94,7 +100,7 @@ def compile(pipeline_config: str, provider_config: str, output_file: str):
 
         compile_fn(pipeline_config_contents, output_file).run(
             pipeline.Pipeline(
-                pipeline_name=pipeline_config_contents['name'],
+                pipeline_name=parse_namespaced_pipeline_name(pipeline_config_contents['name']),
```

Comment:
we need to do this conversion here as `/` isn't a valid character for pipeline name